packages/superset-ui-query/src/api/legacy/fetchExploreJson.ts

```diff
@@ -0,0 +1,28 @@
+import { SupersetClientInterface, SupersetClient, RequestConfig } from '@superset-ui/connection';
+import { QueryFormData } from '../../types/QueryFormData';
+import { LegacyChartDataResponse } from './types';
+
+export interface Params {
+  client?: SupersetClientInterface;
+  method?: 'GET' | 'POST';
+  requestConfig?: Partial<RequestConfig>;
+  url?: string;
+  formData: QueryFormData;
+}
+
+export default function fetchExploreJson({
+  client = SupersetClient,
+  method = 'POST',
+  requestConfig,
+  url = '/superset/explore_json/',
+  formData,
+}: Params) {
+  const fetchFunc = method === 'GET' ? client.get : client.post;
+
+  return fetchFunc({
+    ...requestConfig,
+    // TODO: Have to transform formData as query string for GET
```

Comment:
Do you have an example for what a query string could look like?

Reply:
There is a utility function in `incubator-superset` that has to be ported here.